en/docs/learn/password-recovery-via-user-preferred-notification-channel.md

```diff
@@ -0,0 +1,597 @@
+            "href": "/t/carbon.super/api/users/v1/recovery/password/recover",
+            "type": "POST"
+        }
+      ]
+    },
+    {
+      "mode": "recoverWithChallengeQuestions",
+      "links": [
+        {
+          "rel": "next",
+          "href": "/t/carbon.superidentity/recovery/v0.9/security-question?username=sominda2",
+          "type": "GET"
+        }
+      ]
+    }
+    ]
+    ``` 
+    
+    !!! note
+        The validity period of the recovery code is determined by the 2nd step of 
+        Configuring Password Recovery.  
+        
+    !!! info
+        To tryout password recovery with challenge question, refer [Configuring Password Reset with Challenge 
+        Questions](../../learn/configuring-password-reset-with-challenge/).
+    
+3. Use the `recoveryCode` and a preferred channel `id` to get notifications via that channel.
+
+    ```
+    curl -X POST "https://localhost:9443/api/users/v1/recovery/password/recover" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"recoveryCode\":\"1234-5678-2455-3433\",\"channelId\":\"1\",\"properties\":[{\"key\":\"key\",\"value\":\"value\"}]}"
+    ```      
+    
+4. The API will return the following response upon a successful recovery and the user will be notified 
+via the selected channel.
+
+    ```
+    {
+      "code": "PWR-02001",
+      "message": "Password recovery information sent via user preferred notification channel.",
+      "notificationChannel": "EMAIL",
+      "resendCode": "8dde8fd4-c58d-4408-a835-a9954ebc278a",
+      "links": [
+         {
+            "rel": "next",
+            "href": "/t/carbon.super/api/users/v1/recovery/password/confirm",
+            "type": "POST"
+         },
+         {
+            "rel": "resend",
+            "href": "/t/carbon.super/api/users/v1/recovery/password/resend",
+            "type": "POST"
+         }
+      ]
+    }
+    ```
+    
+5. If you want to resend the notifications to the user via the notified channel in the above step, use the 
+`resendCode` with the resend notifications API.
+
+    ```
+    curl -X POST "https://localhost:9443/api/users/v1/recovery/password/resend" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"resendCode\":\"1234-2ws34-1234\",\"properties\":[{\"key\":\"key\",\"value\":\"value\"}]}"
+    ```        
+
+6. The API will send the following response upon successful notification.
+
+    ```
+    {
+    "code": "UAR-02001",
+    "message": "Confirmation code resent to the user.",
+    "notificationChannel": "EMAIL",
+    "resendCode": "8ebefae5-0a80-4edf-ac2d-6034384e45c0",
+    "links": [
+        {
+            "rel": "next",
+            "href": "/t/carbon.super/api/users/v1/recovery/password/confirm",
+            "type": "POST"
+        },
+        {
+            "rel": "resend",
+            "href": "/t/carbon.super/api/users/v1/recovery/password/resend",
+            "type": "POST"
+        }
+    ]
+    }
+    ```
```

Comment:
This response can be part of the previous step. You can use code tabs to have the request and response side by side.

Reply:
Please apply wherever applicable